string id
fake_laser_sim/Shape shape
---
bool success
string message